kepler/parsers.py

```diff
@@ -120,3 +124,12 @@ def convert_coord(cls, coordinate, precision=10):
             dec = dec * -1
         getcontext().prec = o_precision
         return dec
+
+    @classmethod
+    def pad_034(self, coordinate):
+        c = list(coordinate)
```

Comment:
Strings in python can already be treated as lists, so no need to explicitly do so. I would suggest doing something like this to get the hemisphere and coordinate parts:
```python
h, c = coordinate[0], coordinate[1:]
```